src/PrestaShopBundle/Bridge/AdminController/AdminControllerTrait.php

```diff
@@ -24,60 +24,21 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 
-declare(strict_types=1);
```

Comment:
Why removing it ?

Reply:
unintentional. Recreated the class and forgot. putting back